src/lib.rs

```diff
@@ -170,6 +170,10 @@ pub mod vulkan;
 #[cfg(all(windows, feature = "d3d12"))]
 pub mod d3d12;
 
+// todo(lily): add a feature toggle for metal?
+#[cfg(any(target_os = "macos", target_os = "ios"))]
```

Comment:
```suggestion
#[cfg(all(any(target_os = "macos", target_os = "ios"), feature = "metal"))]
```

This should resolve your TODO comment.

Reply:
Changed it in latest commit